modules/dnn/cmake/download_google_inception5.cmake

```diff
@@ -0,0 +1,30 @@
+function(download_google_inception5 dst_dir status_var)
```

Comment:
No need to download these files, because they are not external to OpenCV.
- space_shuttle.jpg : opencv sources
- tensorflow_inception_graph.pb: opencv_extra testdata directory, check for `$ENV{OPENCV_TEST_DATA_PATH}`